### DISCUSSION TITLE
I could not locate the file to change the sentences (session name) to the Portuguese language (Brazil)

### DISCUSSION THREAD

**Discussion by e-Feitosa** [sircharlo/meeting-media-manager#2401] [General]
I would like to manually change (translate) the titles of the sessions to:

![image](https://github.com/sircharlo/meeting-media-manager/assets/90984283/4876fc84-a74a-453d-b2dc-3e3d2b02afd9)
**FAÇA SEU MELHOR NO MINISTÉRIO**
![image](https://github.com/sircharlo/meeting-media-manager/assets/90984283/b8b754a0-345a-408e-84f2-d1979c0d0111)
**FAÇA SEU MELHOR NO MINISTÉRIO**
![image](https://github.com/sircharlo/meeting-media-manager/assets/90984283/da3de46c-2dba-4296-a3cf-c5a51c7a5654)
**NOSSA VIDA CRISTÃ**

**Comment by mtdvlpr** [comment]
Look for the following folder: `C:\Users\USER\AppData\Roaming\meeting-media-manager\Publications\JW_LANG\mwb`.

* USER = the username of your Windows user
* JW_LANG = The language code for your language (e.g. English is E, Dutch is O).

In it, you should see some folders like `20240300` and `20240100` and maybe a `headings.json` file. If such a file is not present, you should create one.

Inside the file, you need the following content:

```js
{
  "treasures": "FAÇA SEU MELHOR NO MINISTÉRIO",
  "apply": "FAÇA SEU MELHOR NO MINISTÉRIO",
  "living": "NOSSA VIDA CRISTÃ"
}
```

**Comment by mtdvlpr** [comment]
This is the file you want to put inside the `mwb` folder (I filled in the correct values for you):

[headings.json](https://github.com/sircharlo/meeting-media-manager/files/14601077/headings.json)

**Reply by e-Feitosa** [reply]
`Thank you very much for your time and your help.`
I tested and it worked, but we have a small problem here, as I have 3 congregations in M3 and use the Koofr server, I created a folder for each congregation, and in this case, M3 did not do the translation, as a test, created a new congregation and did not make folder on the koofr server, and then the normal translation is done, using the server or not. 
> In short: If you create a folder on the Koofr server, type, congregation 1 (Saturday), Congregation 2 (Sunday) and Congregation 3 (Sunday), the M3 does not translate the sessions of the mid-week meeting.
